nft/ethereum_payment_obligation.go

```diff
@@ -314,7 +333,7 @@ func createProofData(proofspb []*proofspb.Proof) (*proofData, error) {
 		proofs[i] = property
 	}
 
-	return &proofData{Values: values, Salts: salts, Proofs: proofs}, nil
+	return &proofData{Props: props, Values: values, Salts: salts, Proofs: proofs}, nil
```

Comment:
Why are these properties required?

Reply:
@vimukthi-git why are these required?

Reply:
Its required by the smart contract to construct certain properties to verify
you can ask @rdinicut